web/cypress/integration/settings/settings-smoketest.spec.js

```diff
@@ -54,7 +54,7 @@ context('ssl smoke test', () => {
     cy.contains('Settings').click();
     cy.wait(500);
     cy.url().should('contains', '/settings');
-    cy.get('#grafanaURL').clear().type('https://www.baidu.com/');
+    cy.get('#grafanaURL').clear().type('https://www.apiseven.com/');
```

Comment:
no no no, that is a bad name.

we can use `github.com` or `apisix.apache.org`

Reply:
good idea, fixed.